iModelJsNodeAddon/presentation/ECPresentationSerializer.cpp

```diff
@@ -1119,6 +1131,365 @@ rapidjson::Document IModelJsECPresentationSerializer::_AsJson(ContextR ctx, Node
     return json;
     }
 
+/*---------------------------------------------------------------------------------**//**
+* @bsimethod
++---------------+---------------+---------------+---------------+---------------+------*/
+static Utf8String GetECValueTypeName(PrimitiveType type)
+    {
+    switch (type)
+        {
+        case PRIMITIVETYPE_Binary:
+            return "binary";
+        case PRIMITIVETYPE_Boolean:
+            return "boolean";
+        case PRIMITIVETYPE_DateTime:
+            return "dateTime";
+        case PRIMITIVETYPE_Double:
+            return "double";
+        case PRIMITIVETYPE_Integer:
+            return "int";
+        case PRIMITIVETYPE_Long:
+            return "long";
+        case PRIMITIVETYPE_Point2d:
+            return "point2d";
+        case PRIMITIVETYPE_Point3d:
+            return "point3d";
+        case PRIMITIVETYPE_String:
+            return "string";
+        default:
+            return "";

```

Comment:
I'd rather move this `return "";` to the bottom of the function (outside of `switch`)

Reply:
Moved.